packages/pkp-sui/src/lib/TransactionBlockData.ts

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { toB58 } from '@mysten/bcs';
```

Comment:
Instead of introducing a new dependency, could you please change this to use the `bs58` package?